src/decoration.c

```diff
@@ -489,31 +489,31 @@ void sub_8126ABC(void)
 {
     sSecretBasePCMenuCursorPos = 0;
     ScriptContext2_Enable();
-    sub_8126A88();
-    sub_8126C08();
+    SecretBasePC_ShowDecorateMenu();
+    SecretBasePC_DisplayDecorateMenuItemDescription();
 }
 
-void sub_8126AD8(u8 taskId)
+void SecretBasePC_InitSecretBaseDecorateMenuTask(u8 taskId)
 {
     sub_8126ABC();
-    gUnknown_0203A17C.items = gSaveBlock1Ptr->secretBases[0].decorations;
-    gUnknown_0203A17C.pos = gSaveBlock1Ptr->secretBases[0].decorationPos;
-    gUnknown_0203A17C.size = sizeof(gSaveBlock1Ptr->secretBases[0].decorations);
-    gUnknown_0203A17C.isPlayerRoom = FALSE;
-    gTasks[taskId].func = sub_8126B80;
+    sWorkingDecorData.items = gSaveBlock1Ptr->secretBases[0].decorations;
+    sWorkingDecorData.pos = gSaveBlock1Ptr->secretBases[0].decorationPos;
+    sWorkingDecorData.size = sizeof(gSaveBlock1Ptr->secretBases[0].decorations);
+    sWorkingDecorData.isPlayerRoom = FALSE;
+    gTasks[taskId].func = SecretBasePC_DecorateMenuTask;
```

Comment:
The naming convention for tasks is to prefix the function with `Task_`. So in this case, `Task_SecretBasePc_DecorateMenu`.

Reply:
There is no set-in-stone convention for Task function names, and either of these names would do.

Reply:
Yeah, I've seen both in the code. I've seen `Task_DescriptionOfFunction`, but then I've also seen `Concept_DescriptionOfFunctionTask`, and then there's even a few instances of `concept_description_of_function`. It would be nice if there was a more set-in-stone convention...